src/main/resources/static/javascript/services.js

```diff
@@ -210,23 +210,14 @@ angular.module('smartCampUZApp')
             	
             	$http({
                     method: 'GET',
-                    url: 'locationFromCoords',
-				    headers: {
-				        x: x,
-				    	y: y,
-				    	buildingFloors: buildingFloors
-				    }
+                    url: 'locationFromCoords?x=' + x + '&y=' + y + '&buildingFloors=' + buildingFloors,
```

Comment:
This final coma is unnecessary. Moreover, a content-type header must be included in every http request.